frontend/src/scenes/data-warehouse/settings/DataWarehouseSourcesTable.tsx

```diff
@@ -0,0 +1,325 @@
+    const { deleteSource, reloadSource } = useActions(dataWarehouseSettingsLogic)
+
+    const renderExpandable = (source: ExternalDataStripeSource): JSX.Element => {
+        return (
+            <div className="px-4 py-3">
+                <div className="flex flex-col">
+                    <div className="mt-2">
+                        <SchemaTable schemas={source.schemas} />
+                    </div>
+                </div>
+            </div>
+        )
+    }
+
+    if (!dataWarehouseSourcesLoading && dataWarehouseSources?.results.length === 0) {
+        return (
+            <ProductIntroduction
+                productName="Data Warehouse Source"
+                productKey={ProductKey.DATA_WAREHOUSE}
+                thingName="data source"
+                description="Use data warehouse sources to import data from your external data into PostHog."
+                isEmpty={dataWarehouseSources?.results.length == 0}
+                docsURL="https://posthog.com/docs/data-warehouse"
+                action={() => router.actions.push(urls.pipelineNodeDataWarehouseNew())}
+            />
+        )
+    }
+
+    return (
+        <LemonTable
+            dataSource={dataWarehouseSources?.results ?? []}
+            loading={dataWarehouseSourcesLoading}
+            disableTableWhileLoading={false}
+            columns={[
+                {
+                    title: 'Source Type',
+                    key: 'name',
+                    render: function RenderName(_, source) {
+                        return source.source_type
+                    },
+                },
+                {
+                    title: 'Table Prefix',
+                    key: 'prefix',
+                    render: function RenderPrefix(_, source) {
+                        return source.prefix
+                    },
+                },
+                {
+                    title: 'Status',
+                    key: 'status',
+                    render: function RenderStatus(_, source) {
+                        return <LemonTag type={StatusTagSetting[source.status] || 'default'}>{source.status}</LemonTag>
+                    },
+                },
+                {
+                    title: 'Sync Frequency',
+                    key: 'frequency',
+                    render: function RenderFrequency() {
+                        return 'Every 24 hours'
+                    },
+                },
+                {
+                    title: 'Last Successful Run',
+                    key: 'last_run_at',
+                    tooltip: 'Time of the last run that completed a data import',
+                    render: (_, run) => {
+                        return run.last_run_at ? (
+                            <TZLabel time={run.last_run_at} formatDate="MMM DD, YYYY" formatTime="HH:mm" />
+                        ) : (
+                            'Never'
+                        )
+                    },
+                },
```

Comment:
The weekly number matters less because it matters less how much is synced per week and more important to just know how many rows total are synced from that source. We show it on a per schema basis (if you click the dropdown). I've added a "total rows synced metric"

Reply:
The motivation behind the spark-graph is: imagine a user looks at the pipeline tab 1-2 times per week. Did anything fail / odd happen that I should look into?

The way this works for batch exports is: There's a run that operates on top of events - each run can succeed or fail. If the run fails we show event count as failed (and didn't succeed during a rerun if it did it shows up under success), if it succeeds we show the total event count as green. What this means is if something failed 2 days ago and wasn't retried it's immediately obvious for the user & they can then go and re-run them or investigate further. If everything is green then also they know it's all good.

Is there a similar concept for data warehouse (what would a user want to see that gives them an overview of what happened over the past week). Qs:
1. Can some schemas fail to sync while others work?
2. If we hit the row limit, do we know how many rows we didn't sync?

Just an example idea for inspiration:
1. For each day take the latest succeeding sync (because the user doesn't care if a sync failed if a later one succeeds & our systems look more reliable) & show that row count as green - Seeing the row count increase or decrease a lot or not changing on a graph I feel could be more useful for the user compared to just a raw number
2. if no sync succeeded then show the existing row count as red
3. If some partial success/failure can happen show that as green/red accordingly

If there's an easy way to do this, let's, if not maybe sth to consider for later.